cms/startup.py

```diff
@@ -38,3 +42,10 @@ def add_mimetypes():
     mimetypes.add_type('application/x-font-opentype', '.otf')
     mimetypes.add_type('application/x-font-ttf', '.ttf')
     mimetypes.add_type('application/font-woff', '.woff')
+
+
+def startup_notification_subsystem():
+    """
+    Initialize the Notification subsystem
+    """
+    startup.initialize()
```

Comment:
What happens if the initialize() workflow encounters an exception?  Do you need to try/except this?